RELEASE_NOTES.md

```diff
@@ -15,6 +15,10 @@ v1.0.0-beta.x.y
   a `CppPluginSystemManagerPlugin` object.
   [#1115](https://github.com/OpenAssetIO/OpenAssetIO/issues/1115)
 
+- Test harness updated to automatically load C++ plugins if a python
```

Comment:
[consider] Perhaps clarify this is the API Compliance Suite test harness, `openassetio.test.manager`, not our internal unit tests.

Reply:
:+1:  Did this, but am just rebasing and merging, don't think it needs a re-review.